config/index.js

```diff
@@ -274,6 +274,23 @@ var conf = convict({
     format: Boolean,
     env: 'LOCKOUT_ENABLED',
     default: false
+  },
+  oauth: {
+    host: {
+      doc: 'OAuth host for verifying tokens',
+      default: 'localhost',
+      env: 'OAUTH_HOST'
+    },
+    port: {
+      doc: 'OAuth port for veryfing tokens',
+      default: 9010,
+      env: 'OAUTH_PORT'
+    },
+    insecure: {
+      doc: 'Connect to OAuth server via insecure plain http',
+      default: false,
+      env: 'OAUTH_INSECURE'
```

Comment:
nit: I prefer super explicit and verbose names for these turn-off-my-security options; how about something like `OAUTH_ALLOW_INSECURE_CONNECTIONS`?